babel.config.js

```diff
@@ -34,7 +34,10 @@ module.exports = {
           // List of supported proposals: https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#ecmascript-proposals
           proposals: true,
         },
-        exclude: ['es.error.cause'],
+        exclude: [
+          'es.error.cause',
+          process.env.NODE_ENV !== 'test' && 'proposal-dynamic-import',
```

Comment:
Without this conditional, Jest wants us to pass `--experimental-vm-modules` to `node`. I tried that and most of our test suites actually worked save for the CLI and codemods. But I figured it may be easier to just disable this config for testing since we only really have one ES module we're importing in the entire framework